.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Integration Tests
+on:
+  push:
+    branches:
+      - master
+      - develop
+  pull_request:
+    branches:
+      - master
+      - develop
+
+env:
+  REACT_APP_NETWORK_URL: 'https://mainnet.infura.io/v3/4bf032f2d38a4ed6bb975b80d6340847'
```

Comment:
do we want this exposed?

Reply:
It is Uniswap original env, if they don't care, we care less :) 

Also, I wouldn't care even if it's ours, worst case, we need to invalidate it and it makes it easier to run the project.
I agree in this file is not required though, cause it could be a "secret", but since it's also in the ".env" i think there's no need to overcomplicate. Good comment though

Reply:
Yes, we can definitely move the infura key to SECRETS.
But if we don't care...